app/helpers/application_helper.rb

```diff
@@ -1,2 +1,26 @@
 module ApplicationHelper
+	
+	def friendly_age(birthdate)
+		years_old = Date.today - birthdate 
+		years_and_months = years_old.divmod(365)
+		years = years_and_months[0]
+		months = years_and_months[1] / 30
+		months_i = months.to_i
+		if months_i < 1
```

Comment:
This if/elsif/else logic certainly works, but do you think it's easy to follow?

What about an approach where we just put in years and months and then remove both cases of '0 years' and '0 months', which is also what this logic is trying to prevent, right?

For example:

```
friendly_age = "#{years} years, #{months} months"
friendly_age.gsub!('0 years, ', '')
friendly_age.gsub!(', 0 months', '')
```  

What do you think?